forms-flow-nav/src/sidenav/Sidebar.jsx

```diff
@@ -145,7 +147,23 @@ const Sidebar = React.memo(({ props }) => {
       setTenantLogo(logo);
     }
   }, [tenant]);
-
+  
+  useEffect((()=>{
+    const sections = [
+      { key: "0", matchExps: ["formflow", "bundle", "subflow", "decision-table"] },
```

Comment:
1. the key is equal to eventKey  right so instead of passing index we can give meaning full name for that

Reply:
@fahad-aot please do